workflow/rules/malt.smk

```diff
@@ -28,7 +28,7 @@ rule Build_Malt_DB:
         "grep -Ff {output.seqids_project} {params.nt_fasta} | sed 's/>//g' > {output.project_headers}; "
         "seqtk subseq {params.nt_fasta} {output.project_headers} > {output.project_fasta} 2> {log}; "
         "unset DISPLAY; "
-        "malt-build -i {output.project_fasta} -a2t {params.accession2taxid} -s DNA -t {threads} -d {output.db} &>> {log}"
+        "malt-build -i {output.project_fasta} -a2taxonomy {params.accession2taxid} -s DNA -t {threads} -d {output.db} &>> {log}"
```

Comment:
I wonder if there is a option mismatch between malt versions? For malt 0.53 the help string states
`       -a2t, --acc2taxa [string]            Accession-to-Taxonomy mapping file.
	`

Reply:
I replaced "malt" by "hops" as it gives you automatically MaltExtract installed, and -a2t was not recognized by the malt-run version included in hops, but instead -a2taxonomy worked

Reply:
Ok, then I think it is as I said. I just tried installing hops which pulls in the dependency malt=0.41. The help string is:
`-a2taxonomy, --acc2taxonomy [string]   Accession-to-Taxonomy mapping file.`

Is maltextract the only package we need from hops?